tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -979,12 +980,12 @@ void EnqueueProgramCommand::assemble_device_commands(
                 this->packed_write_max_unicast_sub_cmds,
                 multicast_go_signals_payload);
         }
-
         for (KernelGroup& kernel_group : program.get_kernel_groups(CoreType::ETH)) {
             kernel_group.launch_msg.kernel_config.mode = DISPATCH_MODE_DEV;
             kernel_group.launch_msg.kernel_config.dispatch_core_x = this->dispatch_core.x;
             kernel_group.launch_msg.kernel_config.dispatch_core_y = this->dispatch_core.y;
             kernel_group.launch_msg.kernel_config.kernel_config_base = eth_l1_kernel_config_base;
+            kernel_group.launch_msg.kernel_config.host_assigned_op_id = program.get_global_id();
```

Comment:
Can you please try find a better name instead of op_id?

It is minor, but still leaks the TT-NN concept into metal

Maybe something like 
kernel_config.program_group_id

And id consider renaming get_global_id to something like program.get_group_id

Reply:
I just dropped the op and changed it `host_assigned_id`. I believe that is good representation of what this variable is. It is all for post proc association so host can assign different ids depending on the flow. 


Program is a host concept already so host_assigned_id would be awkward. 
How is runtime_id? This id is different in that two different runtime instance of the same program are given different IDs